From 056387ba9f0d3d41a86db64b2a1e5d3df3257c1e Mon Sep 17 00:00:00 2001
From: Alison Wang <alison.wang@nxp.com>
Date: Thu, 19 Jul 2018 17:38:52 +0800
Subject: [PATCH 275/706] drm: hdp: Increase the pixel clock to 594MHz

As 3840x2160p60 is supported, the pixel clock can be 594MHz. The patch
increases the pixel clock to 594MHz for common modes.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
(cherry picked from commit 935b47b079e396b14e9318b5e52bec64bd4c6963)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-hdp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-hdp.c b/drivers/gpu/drm/imx/hdp/imx-hdp.c
index 0cb18b014d06..faf2ed62369f 100644
--- a/drivers/gpu/drm/imx/hdp/imx-hdp.c
+++ b/drivers/gpu/drm/imx/hdp/imx-hdp.c
@@ -566,7 +566,7 @@ imx_hdp_connector_mode_valid(struct drm_connector *connector,
 {
 	enum drm_mode_status mode_status = MODE_OK;
 
-	if (mode->clock > 150000)
+	if (mode->clock > 594000)
 		return MODE_CLOCK_HIGH;
 
 	return mode_status;
-- 
2.17.1

